### PR TITLE
Specify explicit priorities for all form type extensions

### LINKED ISSUE #13450: [CoreBundle] form type extensions should specify a priority
I tried to remove a field added by a `FormTypeExtension` in the CoreBundle (namely the `themeName` field in `ChannelType` form), and autowiring the extension just didn't cut it: I had to specifically use a negative priority to ensure that my extension applies its effect after the core one.

Thus I think it would make sense for CoreBundle FormTypeExtensions to specify an explicit positive priority. This would ensure that app extensions are always applied after the Core ones.

I can provide a PR, just tell me what priority to use for CoreBundle extensions. Is 10 ok?

### PR DESCRIPTION
**Pull request by rimas-kudelis**
| Q               | A
| --------------- | -----
| Branch?         | 1.11
| Bug fix?        | no
| New feature?    | yes
| BC breaks?      | potentially
| Deprecations?   | no
| Related tickets | fixes #13450 
| License         | MIT

I'm making the PR against 1.11, because it hasn't been released yet, but you're free to retarget it against master if you don't feel like accepting it into the release train this late.

### CODE REVIEW

File: UPGRADE-1.11.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ This obvious dev dependency was part of Sylius requirements. In 1.11 we've moved
 
 Due to the drop of support PHP `7.4` Sylius also will not support it since version `1.11`.
 
-### Potential BC-break
+### Potential BC-break: state machine callbacks
```
Comment (vvasiloi) [inline UPGRADE-1.11.md:23]: Can you sub-sections to `Potential BC-break` instead, like this:

```markdown
### Potential BC-break

#### State machine callbacks

#### Form type extensions
```

Similar to https://github.com/Sylius/Sylius/pull/13531/files.

**Review by vvasiloi — APPROVED** [review]

File: UPGRADE-1.11.md (hunk 1 above)
Comment (rimas-kudelis) [reply UPGRADE-1.11.md]: Done

File: UPGRADE-1.11.md [hunk 2]
```diff
@@ -20,7 +20,9 @@ This obvious dev dependency was part of Sylius requirements. In 1.11 we've moved
 
 Due to the drop of support PHP `7.4` Sylius also will not support it since version `1.11`.
 
-### Potential BC-break
+### Potential BC-breaks
+
+### State machine callbacks
```
Comment (vvasiloi) [inline UPGRADE-1.11.md:25]:
```suggestion
#### State machine callbacks
```

Comment (rimas-kudelis) [reply UPGRADE-1.11.md]: :man_facepalming:

**Review by Roshyo — APPROVED** [review]

**Review by lchrusciel — APPROVED** [review]
Part of me wants to merge it to master, but on the other hand with 1.11 we will have feedback and benefits from this change much faster.

**Review by Zales0123 — APPROVED** [review]

**Comment by Zales0123** [comment]
@rimas-kudelis I've allowed myself to rebase and resolve conflict, it's mergeable when the build passed. Thanks! 🚀

**Comment by rimas-kudelis** [comment]
@Zales0123 Very good, very good. :smile: I didn't notice the PR wasn't mergeable anymore.

**Comment by Zales0123** [comment]
Thanks, Rimas! :1st_place_medal:

**Pull request merged**